src/sentry/static/sentry/app/components/links/listLink.tsx

```diff
@@ -10,16 +10,15 @@ type DefaultProps = {
   activeClassName: string;
 };
 
-type Props = DefaultProps & {
-  to: LocationDescriptor;
-  className?: string;
-  query?: string;
-  onClick?: () => void;
-  // If supplied by parent component, decides whether link element
-  // is "active" or not ... overriding default behavior of strict
-  // route matching
-  isActive?: (location: LocationDescriptor, indexOnly?: boolean) => boolean;
-};
+type Props = DefaultProps &
```

Comment:
in this case, I would remove "activeClassName" from the defaultProps as it is already part of the Link props

Reply:
I think this is _okay_, I need `activeClassName` in the `DefaultProps` type, since I need to type `defaultProps`.

I could probably omit it, but this is fine, typescript merges these without any problems.